stdlib/2and3/bisect.pyi

```diff
@@ -1,12 +1,14 @@
+# Stubs for bisect
+
 from typing import Sequence, TypeVar
 
 _T = TypeVar('_T')
 
-def insort_left(a: Sequence[_T], x: _T, lo: int = ..., hi: int = ...): pass
-def insort_right(a: Sequence[_T], x: _T, lo: int = ..., hi: int = ...): pass
 
-def bisect_left(a: Sequence[_T], x: _T, lo: int = ..., hi: int = ...): pass
-def bisect_right(a: Sequence[_T], x: _T, lo: int = ..., hi: int = ...): pass
+def bisect_left(a: Sequence[_T], x: _T, lo: int = ..., hi: int = ...) -> int: ...
```

Comment:
It's perfectly legal for `x` to not be in `a`.

For example,

`bisect.bisect_left([1,2,3], None)`

is valid code. (And bisect_left will just return `0` in this case)

So the type parameter here is overkill.